src/Authentication/Authentication/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+namespace Microsoft.Graph.PowerShell.Authentication.Extensions
+{
+    using System;
+    public static class StringExtensions
+    {
+        /// <summary>
+        /// Indicates whether a specified string is null, empty, consists only of white-space, or has the specified search value.
+        /// </summary>
+        /// <param name="target">The target string to look in.</param>
+        /// <param name="searchValue">The substring to seek.</param>
+        /// <param name="comparison">The <see cref="StringComparison"/> to use. This defaults to <see cref="StringComparison.OrdinalIgnoreCase"/>.</param>
+        /// <returns>true if the searchValue parameter occurs within this string; otherwise false.</returns>
+        public static bool ContainsNotNull(this string target, string searchValue, StringComparison comparison = StringComparison.OrdinalIgnoreCase)
```

Comment:
this seems oddly named since it returns true only if the value matches.  not sure why you need it but perhaps you just wanted to combine null checks.  seems like isMatch or something is a more appropriate name